src/scenes/Projects/Budget/ProjectBudget.tsx

```diff
@@ -0,0 +1,176 @@
+    variables: { id: projectId },
+  });
+  const [updateBudgetRecord] = useUpdateProjectBudgetRecordMutation();
+
+  const projectName = data?.project.name.value;
+  const canReadBudget = data?.project.budget.canRead;
+  const canEditBudget = data?.project.budget.canEdit;
+  const budget = data?.project.budget.value;
+  const budgetRecords = budget?.records ?? [];
+
+  interface BudgetRowData {
+    id: string;
+    organization: string;
+    fiscalYear: string;
+    amount: string;
+    canEdit: boolean;
+  }
+
+  const rowData = budgetRecords.reduce((rows: BudgetRowData[], record) => {
+    const { amount, fiscalYear, id, organization } = record;
+    const { value: dollarAmount, canEdit } = amount;
+    const { value: year } = fiscalYear;
+    const orgName = organization.value?.name.value;
+
+    const row = {
+      id,
+      organization: orgName ?? '',
+      fiscalYear: String(year),
+      amount: String(dollarAmount),
+      canEdit,
+    };
+    return rows.concat(row);
+  }, []);
+
+  const columns = [
+    {
+      title: 'ID',
+      field: 'id',
+      hidden: true,
+    },
+    {
+      title: 'Organization',
+      field: 'organization',
+      editable: 'never' as const,
+    },
+    {
+      title: 'Fiscal Year',
+      field: 'fiscalYear',
+      editable: 'never' as const,
+      render: (rowData: BudgetRowData) => `FY${rowData.fiscalYear}`,
+    },
+    {
+      title: 'Amount',
+      field: 'amount',
+      type: 'currency' as const,
+      editable: (_: unknown, rowData: BudgetRowData) => rowData.canEdit,
+      cellStyle: {
+        display: 'flex',
+        flexDirection: 'row' as const,
+        justifyContent: 'flex-end',
+      },
+    },
+    {
+      title: 'Can Edit',
+      field: 'canEdit',
+      hidden: true,
+    },
+  ];
+
+  return (
+    <Content>
+      {error ? (
+        <Typography variant="h4">Error fetching Project Budget</Typography>
+      ) : canReadBudget === false ? (
+        <Typography variant="h4">
+          You do not have permission to view this project's budget
+        </Typography>
+      ) : (
+        <>
+          {loading ? (
+            <Skeleton variant="text" width="20%" />
+          ) : (
+            <Breadcrumbs>
+              <Breadcrumb to="/projects">Projects</Breadcrumb>
+              <Breadcrumb to={`/projects/${projectId}`}>
+                {projectName}
+              </Breadcrumb>
+            </Breadcrumbs>
+          )}
+          <header className={classes.header}>
+            {loading ? (
+              <>
+                <Skeleton variant="text" width="20%" />
+                <Skeleton variant="text" width="10%" />
+              </>
+            ) : (
+              <>
+                <Typography variant="h2">
+                  {data?.project.name.value} Budget
+                </Typography>
+                {budget && (
+                  <Typography variant="body1" className={classes.total}>
```

Comment:
Could we use a standard variant here instead of a manual `em` value? Maybe h3/h4?